.github/PROCESS.md

```diff
@@ -16,14 +16,35 @@ Keep guides short: we don't create rules; we automate.
 2. Pull request **title** should be accurate and descriptive as it is used in the generated changelog.
    It should not end with a dot.
    It should not mention the issue number but describe the fixed issue instead.
-3. Pull request **description** typically just mentions the issue number like "Refs #123." or "Closes #456.".
+3. Pull request **description** must mention the issue number like "Refs #123." or "Closes #456.".
    Do not use `https://github.com/org/repo/issue/123` even if it is rendered the same on GitHub.
+   We provide a pull requests template that includes suggestions and readiness checklist.
    End sentences with a dot.
 4. Commit messages (both titles and bodies) are not important and can be "WIP" or anything else.
    All commits are always squashed on merge by GitHub, so there is no need to squash them manually, amend them, and/or do force pushes.
-   Autogenerated GitHub's squash commit's title and body should be manually replaced by PR's title and description; that includes both manual merges and auto-merges.
+   Autogenerated GitHub's squash commit's title should be manually replaced by PR's title.
+   Autogenerated GitHub's squash commit's body should be manually replaced by "Refs #{issue_number}". 
```

Comment:
```suggestion
   Autogenerated GitHub's squash commit's body should be manually replaced by "Refs #{issue_number}" or "Closes #{issue_number}". 
```

Reply:
No here is a list with explanations of differences in between them:
* Closes closes issue automatically
* Refs mentions without closing the issue.

Reply:
There is no mention of closing issues or whatever

Reply:
> There is no mention of closing issues or whatever

But such questions will appear inevitably.
And when GitHub changes that behaviour we need to change it also.